src/app/services/culture-service.service.ts

```diff
@@ -0,0 +1,50 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Environment, MockRoutes } from 'src/constants/routes';
+
+export interface details {
+  id : number;
+  title : string;
+  category : string;
```

Comment:
We are not writing interface in service , see where existing interfaces are written

Reply:
is it in card.ts sir?

Reply:
solved this have put this interface inside model/card.ts